decentraland/unity-renderer
# feat: mouse Y invert setting

## What does this PR change?

Fixes #1899 
Adds mouse Y invert setting in the general settings panel

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/invert-y-config
2. Verify that first and third person camera are working as expected
3. Go into general settings
4. Verify that invert mouse Y is off
5. Enable it
6. Verify the inverse Y input in first and third person
7. Exit and re-enter decentraland to verify that the setting has been saved

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
